internal/core/src/exec/expression/TermExpr.cpp

```diff
@@ -137,9 +138,18 @@ PhyTermFilterExpr::CanSkipSegment() {
         max = i == 0 ? val : std::max(val, max);
         min = i == 0 ? val : std::min(val, min);
     }
+    auto can_skip = [&]() -> bool {
```

Comment:
is there a way we can skip by chunk instead of skip by segment?

Reply:
> is there a way we can skip by chunk instead of skip by segment?

yes, we can check if a chunk can be skipped in search_pks function for this case.